src/librustc_driver/lib.rs

```diff
@@ -123,10 +123,6 @@ pub trait Callbacks {
     }
 }
 
-pub struct DefaultCallbacks;
```

Comment:
Clippy uses this: https://github.com/rust-lang/rust-clippy/blob/63f818e67dd4a0365d1dc41b21b50b9cf2d16186/src/driver.rs#L390

Reply:
Couldn't clippy just use an own struct instead? The trait itself is `pub`

Reply:
Sound plausible. Let's me try it.